assets/pmgbootstrap/_theme.scss

```diff
@@ -15,27 +17,152 @@
     padding-bottom: 20px;
 }
 
+@mixin topbar-not-side-nav {
+
+    body > .body {
+        min-height: inherit;
+    }
+
+    .main-content {
+        @include container-fixed;
+        @include main-container();
+        padding-left: 0;
+        padding-right: 0;
+
+        @media (min-width: $screen-sm-min) {
+            width: $container-sm;
+        }
+        @media (min-width: $screen-md-min) {
+            width: $container-md;
+        }
+        @media (min-width: $screen-lg-min) {
+            width: $container-lg;
+        }
+        & > .container {
+
+        }
+        & > .container-fluid {
+
+        }
+    }
+
+}
+
 html, body {
     font-weight: 300;
     background-color: $gray-lighter;
 }
 
+
 body {
-    & > .container {
-        @include main-container();
+    .navbar {
+        padding: 10px;
     }
-    & > .container-fluid {
-        @include main-container-fluid();
+    .main-content, .main, .main-wrap {
+        .breadcrumb {
+            background-color: transparent;
+            text-transform: lowercase;
+        }
     }
+}
 
-    .main, .main-wrap {
-        & > .container {
-            @include main-container();
+html:not(.sidenav) body {
+    @include topbar-not-side-nav;
+}
+
+@media (max-width: $sidenavbar-min-width) {
+    @include topbar-not-side-nav;
+}
+
+
+@media (min-width: $sidenavbar-min-width) {
+    html.sidenav body > .body {
+        display: flex;
+        min-height: 100vh;
+        & > .navbar {
+            flex-grow: 0;
+            flex-shrink: 0;
+            width: 250px;
+            background-color: $navbar-inverse-bg;
+            border-color: transparent;
+            padding: 0px;
+            border-radius: 0px;
+            margin-bottom: 0px;
+            @media (min-width: 768px) {
+                .navbar-nav > li {
+                    float: none;
+                }
+                .navbar-nav.pull-right {
+                    float: left !important;
+                }
+            }
+
+            .navbar-header {
+                padding-top: 15px;
+            }
+
+            .navbar-brand {
+                color: #FFF;
+            }
+            li > a {
+                color: #B7C0CD;
+            }
+            li.active > a {
+                background-color:inherit;
+                color: #FFF;
+            }
+            .dropdown {
+                .caret {
+                    display: none;
+                }
+                .dropdown-menu {
+                    display: none;
+                }
+                &.active, &.open {
```

Comment:
```
&.active,
&.open {
  //css
}
```

I don't think there are any standards in our wiki for this, it's more just my personal preference because I find it easier to read